packages/ui/src/components/header/header.data.js

```diff
@@ -1,133 +1,9 @@
-      {
-        title: 'User Group Home',
-        link: 'https://tug.tidb.io',
-      },
-      {
-        title: 'MVA',
-        link: 'https://tug.tidb.io/MVA',
-      },
-    ],
-  },
-  {
-    title: 'Developer Group',
-    items: [
-      {
-        title: 'DevGroup Home',
-        link: 'https://contributor.tidb.io',
-      },
-      {
-        title: 'Contribution',
-        items: [
-          {
-            title: 'Source Code Learning',
-            link: 'https://pingcap.com/blog-cn/#TiDB-%e6%ba%90%e7%a0%81%e9%98%85%e8%af%bb',
-          },
-          {
-            title: 'Talent Plan',
-            link: 'https://university.pingcap.com/talent-plan/',
-          },
-        ],
-      },
-      {
-        title: 'People',
-        items: [
-          {
-            title: 'Maintainers',
-            link: 'https://contributor.tidb.io/people/maintainer',
-          },
-          {
-            title: 'Committer',
-            link: 'https://contributor.tidb.io/people/committer',
-          },
-          {
-            title: 'Reviewer',
-            link: 'https://contributor.tidb.io/people/reviewer',
-          },
-          {
-            title: 'Active Contributor',
-            link: 'https://contributor.tidb.io/people/active-contributor',
-          },
-          {
-            title: 'Contributor',
-            link: 'https://contributor.tidb.io/people/contributor',
-          },
-        ],
-      },
-      {
-        title: 'Special Interest Group (SIG)',
-        link: 'https://contributor.tidb.io/SIG/',
-      },
-    ],
-  },
-  {
-    title: 'Learning',
-    items: [
-      {
-        title: 'TiDB DBA Courses',
-        link: 'https://university.pingcap.com/',
-      },
-      {
-        title: 'Talent Plan Courses',
-        link: 'https://university.pingcap.com/talent-plan/',
-      },
-      {
-        title: 'PingCAP Certified TiDB Associate (PCTA)',
-        link: 'https://university.pingcap.com/certificate/PCTA/',
-      },
-      {
-        title: 'PingCAP Certified TiDB Professional (PCTP)',
-        link: 'https://university.pingcap.com/certificate/PCTP/',
-      },
-    ],
-  },
-  {
-    title: 'Events',
-    link: 'https://contributor.tidb.io/events',
-  },
-  {
-    title: 'Resources',
-    items: [
-      {
-        title: 'Forum',
-        link: 'https://asktug.com',
-      },
-      {
-        title: 'Slack',
-        link: 'https://slack.tidb.io/invite?team=tidb-community&channel=everyone',
-      },
-      {
-        title: 'Docs',
-        link: 'https://docs.pingcap.com/tidb/stable',
-      },
-      {
-        title: 'Blogs',
-        link: 'https://asktug.com/c/blog/l/latest',
-      },
-      {
-        title: 'Mailing Lists',
-        link: 'https://lists.tidb.io/g/main/subgroups',
-      },
-    ],
-  },
-  {
-    title: 'Job Board',
-    link: 'https://pingcap.com/community-cn/careers/join/',
-  },
-];
+import * as dataEn from './header.data-en';
+import * as dataCn from './header.data-cn';
```

Comment:
I am not quite sure whether is a good practice to mix-use camelCase & kebab-case for filenames. But as a temporary solution before introducing a solid i18n approach, it should be all good. No change is required.